ceph-volume-test/config/definitions/ceph-volume-test.yml

```diff
@@ -33,186 +33,412 @@
-              current-parameters: true
-              predefined-parameters: |
-                SCENARIO=centos7-filestore-create
-                SUBCOMMAND=lvm
-            - name: ceph-volume-scenario
-              current-parameters: true
-              predefined-parameters: |
-                SCENARIO=xenial-bluestore-dmcrypt
-                SUBCOMMAND=lvm
-            - name: ceph-volume-scenario
-              current-parameters: true
-              predefined-parameters: |
-                SCENARIO=xenial-filestore-dmcrypt
-                SUBCOMMAND=lvm
-            - name: ceph-volume-scenario
-              current-parameters: true
-              predefined-parameters: |
-                SCENARIO=centos7-bluestore-dmcrypt
-                SUBCOMMAND=lvm
-            - name: ceph-volume-scenario
-              current-parameters: true
-              predefined-parameters: |
-                SCENARIO=centos7-filestore-dmcrypt
-                SUBCOMMAND=lvm
+      # centos8 based jobs for octopus & master
       - conditional-step:
           condition-kind: shell
           condition-command: |
             #!/bin/bash
             set -x
-            # if the target branch is luminous or mimic we run these tests.
-            if [[ ! "$CEPH_BRANCH" =~ luminous|mimic ]]; then
+            if [[ ! "$CEPH_BRANCH" =~ master|octopus ]]; then
               exit 1
             fi
           on-evaluation-failure: dont-run
           steps:
+          - multijob:
+              name: 'testing ceph-volume lvm'
+              condition: SUCCESSFUL
+              projects:
+                - name: ceph-volume-scenario
+                  current-parameters: true
+                  predefined-parameters: |
+                    SCENARIO=centos8-bluestore-create
+                    SUBCOMMAND=lvm
+                - name: ceph-volume-scenario
+                  current-parameters: true
+                  predefined-parameters: |
+                    SCENARIO=centos8-filestore-create
+                    SUBCOMMAND=lvm
+                - name: ceph-volume-scenario
+                  current-parameters: true
+                  predefined-parameters: |
+                    SCENARIO=centos8-bluestore-dmcrypt
+                    SUBCOMMAND=lvm
+                - name: ceph-volume-scenario
+                  current-parameters: true
+                  predefined-parameters: |
+                    SCENARIO=centos8-filestore-dmcrypt
+                    SUBCOMMAND=lvm
+                - name: ceph-volume-scenario
+                  current-parameters: true
+                  predefined-parameters: |
+                    SCENARIO=centos7-bluestore-prepare_activate
+                    SUBCOMMAND=lvm
+                - name: ceph-volume-scenario
+                  current-parameters: true
+                  predefined-parameters: |
+                    SCENARIO=centos7-filestore-prepare_activate
+                    SUBCOMMAND=lvm
```

Comment:
Where do these scenarios come from? Not sure they actually exist. Also `s/centos8/centos8/`